src/Build/Evaluation/ProjectParser.cs

```diff
@@ -515,12 +516,19 @@ private ProjectImportElement ParseProjectImportElement(XmlElementWithLocation el
                 );
 
             ProjectXmlUtilities.VerifyThrowProjectAttributes(element, s_validAttributesOnImport);
-
             ProjectXmlUtilities.VerifyThrowProjectRequiredAttribute(element, XMakeAttributes.project);
-
             ProjectXmlUtilities.VerifyThrowProjectNoChildElements(element);
 
-            return new ProjectImportElement(element, parent, _project);
+            SdkReference sdk = null;
+            if (element.HasAttribute(XMakeAttributes.sdk))
+            {
+                sdk = new SdkReference(
+                    ProjectXmlUtilities.GetAttributeValue(element, XMakeAttributes.sdk, nullIfNotExists: true),
+                    ProjectXmlUtilities.GetAttributeValue(element, XMakeAttributes.sdkVersion, nullIfNotExists: true),
```

Comment:
So explicit attributes instead of fancy string that needs to be parsed?

The XSD should probably be updated at some point, since SDKs introduced a couple of new attributes / elements.